CONTRIBUTING.md

```diff
@@ -1 +1,93 @@
-# How to contribute [Coming Soon!]
+# How to contribute to Spectral
+
+First of all, thanks for considering contributing to Spectral! ✨ It's people like you that make tools like Spectral awesome. 💖
+
+At Stoplight, we want contributing to Spectral to be an enjoyable and educational project for everyone. Contributions go beyond commits in pull requests. We are excited to receive contributions in the form of:
+
+- feature ideas
+- pull requests
+- triaging issues
+- reviewing pull requests
+- implementations of Spectral in your own projects
+- blog posts, talks referencing the project, tweets
+- and much more!
```

Comment:
not sure you want to leave this open-ended like that

Reply:
I do! The next line makes it even clearer. 

> If it is related to Spectral, we consider it a contribution.